src/server/db_slice.cc

```diff
@@ -571,6 +576,7 @@ void DbSlice::FlushSlotsFb(const SlotSet& slot_ids) {
     SlotId sid = ClusterConfig::KeySlot(key);
     if (slot_ids.contains(sid) && it.GetVersion() < next_version) {
       PerformDeletion(it, shard_owner(), db_arr_[0].get());
+      SendInvalidationTrackingMessage(key);
```

Comment:
Maybe there is some way to embed SendInvalidationTrackingMessage into PerformDeletion and PostUpdate? We call `SendInvalidationTrackingMessage` 10 times and I'm not sure that's all cases covered